frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.styled.tsx

```diff
@@ -165,16 +164,16 @@ export const Footer = styled.footer<{ variant: FooterVariant }>`
   align-items: center;
   ${props => footerVariantStyles[props.variant]}
   height: calc(50 / 16 * 1rem);
-  padding: 0 0.5rem;
+  padding: 0 1em;
```

Comment:
Base the value on 

https://github.com/metabase/metabase/blob/b54bc40cfd54ab24a693efc117fe601bed7c8d8b/frontend/src/metabase/styled-components/layout/FullWidthContainer.tsx#L8-L22